db/migrate/20240124011436_remove_hero_image_idx_on_rooms.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveHeroImageIdxOnRooms < ActiveRecord::Migration[7.1]
+  def change
+    # Removes unneeded index that strong migrations forces us to add when
+    # initially adding the reference/foreign key
+    remove_index :rooms, :hero_image_id
```

Comment:
@zspencer Strong Migrations complained about performing the first migration when I tried to remove the part which adds an index. I tried a few different permutations of `add_reference` and `add_foreign_key` to no avail. But as you rightly pointed out, an index is unnecessary, hence this second migration. Maybe I missed something.

Reply:
Could you push that up? I would not write a second migration and instead use `safety_assured` in the first migration if Strong Migrations gets cranky; since choosing not to add an index for a reference is reasonable in this case; even though it's not normally.

Reply:
Ahh, that is nifty. I should have expected there to be an escape hatch. See latest.